locales/en-US/webextension.properties

```diff
@@ -1,9 +1,10 @@
-addonDescription = Firefox Screenshots takes clips and screenshots from pages, and can save a permanent copy of a page.
-addonAuthorsList = Ian Bicking, Donovan Preston, and Bram Pitoyo
+addonDescription = Firefox Screenshots takes clips and screenshots from the Web and lets you save them temporarily or permanently.
+addonAuthorsList = Ian Bicking, Donovan Preston, Jared Hirsch, Danny Coates, Morpheus Chen, Helen Huang, Mark Liang, Fang Shih, John Gruen, Bram Pitoyo, Sharon Bautista and Sean Martell, Michelle Heubusch
```

Comment:
What about just `Mozilla <https://mozilla.org>`, like we do with Test Pilot's package.json?